# no-empty-args

<!-- RULE-CATEGORIES:START -->

**Included in these configurations**: <ul><li>recommended</li><li>csf</li></ul>

<!-- RULE-CATEGORIES:END -->

## Rule Details

A story should not have an empty args property.

Examples of **incorrect** code for this rule:

```js
// fill me in
```

Examples of **correct** code for this rule:

```js
// fill me in
```

### Options

If there are any options, describe them here. Otherwise, delete this section.

## When Not To Use It

Give a short description of when it would be appropriate to turn off this rule. If not applicable, delete this section.

## Further Reading

If there are other links that describe the issue this rule addresses, please include them here in a bulleted list. Otherwise, delete this section.
